libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/compatibility/DirectionsRouteFeatureCompatibility.kt

```diff
@@ -0,0 +1,87 @@
+package com.mapbox.navigation.base.internal.compatibility
```

Comment:
I forgot I pushed it here. Let's remove it for now and push the associated changes into a separate branch.